scripts/install-web.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable unicorn/no-process-exit */
```

Comment:
Can you add the copyright notice here like we have for the other JS files in the repo?